api/query/cache/service/main.go

```diff
@@ -158,12 +159,25 @@ func main() {
 	var netClient = &http.Client{
 		Timeout: time.Second * 5,
 	}
+
+	// Initializes Logger
+	var gcClient *gclog.Client
+
+	gcClient, err = gclog.NewClient(context.Background(), *projectID)
+	if err != nil {
+		logrus.Fatalf("Failed to initiate gclog: %v", err)
+	}
+	defer gcClient.Close()
+
+	childrenLogger := gcClient.Logger("request_logs", gclog.CommonResource(&monitoredResource))
```

Comment:
Nit: probably better to call it `childLogger`. Also, let's use `request_log_entries` for consistency with the current setup (note the current `childLogger` logs to `request_log_entries`).
```suggestion
	childLogger := gcClient.Logger("request_log_entries", gclog.CommonResource(&monitoredResource))
```